lib/code_sponsor_web/views/template_view.ex

```diff
@@ -0,0 +1,10 @@
+defmodule CodeSponsorWeb.TemplateView do
+  use CodeSponsorWeb, :view
+
+  import CodeSponsorWeb.TableView
+
+  def title(:index), do: "Code Sponsor | Templates"
```

Comment:
Actually, I think all of these can get pushed into a single function that you just pass the View type into.